### PR TITLE
Fix tests for cropped data

### PR DESCRIPTION
**Pull request by victoria-cherkas**
Remove hardcoded cell number assertion in NETCDF/GRIB tests.

### CODE REVIEW

**Comment by victoria-cherkas** [comment]
launch jenkins

**Comment by victoria-cherkas** [comment]
launch jenkins

**Comment by victoria-cherkas** [comment]
launch jenkins

File: iconarray/tests/test_grib.py [hunk 1]
```diff
@@ -66,16 +76,22 @@ def test_grid_edge():
...
-    ds_cell, _ds_edge = _open_file(f_alldata)
 
+    Parameters
+    ----------
+    ds_cell : xr.Dataset
+        dataset containing only variables defined on the grid cell center.
+    """
     ds_cellvars = iconarray.combine_grid_information(ds_cell, f_grid)
 
+    ds_grid = iconarray.open_dataset(f_grid)
```
Comment (AnnikaLau) [inline iconarray/tests/test_grib.py:93]: Shouldn't this be part of a main function or when is this being executed?

File: iconarray/tests/test_grib.py [hunk 2]
```diff
@@ -34,23 +35,32 @@ def _open_file(data):
     return ds_cell, ds_edge
 
 
-def test_grid_edge():
+ds_cell, ds_edge = _open_file(f_alldata)
+
+
+@pytest.mark.parametrize("ds_edge", [(ds_edge)])
```
Comment (cfkanesan) [inline iconarray/tests/test_grib.py:41]: The parametrize is not needed in this case.

Comment (victoria-cherkas) [reply iconarray/tests/test_grib.py:41]: Yeah I wasn't sure, I had tried to pass ds_edge as an argument without the parametrize but this had failed.

**Comment by victoria-cherkas** [comment]
launch jenkins

File: iconarray/tests/test_grib.py (hunk 1 above)
Comment (victoria-cherkas) [reply iconarray/tests/test_grib.py:102]: This is used below 
```
assert (len(ds_cellvars.cell.values) == ds_grid.dims["cell"])
```

File: iconarray/tests/test_grib.py [hunk 3]
```diff
@@ -34,23 +35,32 @@ def _open_file(data):
     return ds_cell, ds_edge
 
 
-def test_grid_edge():
+ds_cell, ds_edge = _open_file(f_alldata)
```
[the remark this answers is not in the document]
Comment (victoria-cherkas) [reply iconarray/tests/test_grib.py]: Done

File: iconarray/tests/test_grib.py (hunk 2 above)
Comment (victoria-cherkas) [reply iconarray/tests/test_grib.py]: Removed

**Comment by victoria-cherkas** [comment]
launch jenkins

**Review by cfkanesan — APPROVED** [review]

**Pull request merged**